pkg/default.nix

```diff
@@ -65,6 +66,8 @@ in
 runCommandLocal "nix-wallpaper"
   rec {
     inherit width height;
+    logoX = logoOffset.x || 0;
+    logoY = logoOffset.y || 0;
```

Comment:
The `||` operator only works on booleans, you want `or`

Reply:
Good catch! Thx. I had to do too much TS lately and my muscle memory got the better of me :/